src/components/Card.js

```diff
@@ -18,7 +27,7 @@ export default class Card {
     this._cardElement
       .querySelector("#card-trash-button")
       .addEventListener("click", () => {
-        this._handleTrashIcon();
+        this.handleDelete(this);
       });
 
```

Comment:
Please, reuse `this._cardLikeButton` in line 20 and 41, to avoid searching the like button again

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.